### PR TITLE
Nav animation

### PR DESCRIPTION
**Pull request by josh-hdz**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -154,17 +151,39 @@ nav {
...
-nav a {
-    display: inline-block;
-    font-size: 32px;
-    font-weight: 700;
-    padding: 1% 1%;
+nav ul{
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    padding: 1em 0;
```
Comment (jmuzy-sps) [inline portfolio/src/main/webapp/style.css]: nit: here and in the rest of this file, alphabetize

**Review by jmuzy-sps — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (josh-hdz) [reply portfolio/src/main/webapp/style.css]: This always gets me and I always have it in mind until I forget hahahaha

Comment (josh-hdz) [reply portfolio/src/main/webapp/style.css]: This always gets me and I always have it in mind until I forget hahahaha

**Comment by josh-hdz** [comment]
I pushed a commit for the alphabetical order. It's not showing up for me in the pull request, but it is showing up in the GitHub branch. You make have the same issue.

**Review by jmuzy-sps — APPROVED** [review]
Like you mentioned, I'm not seeing the alphabetical change, but no worries, I trust that you got it in :)

**Pull request merged**